cypress/support/fragments/inventory/inventorySearch.js

```diff
@@ -175,6 +175,12 @@ export default {
     cy.do(Button('Search').click());
     // cy.wait(['@getHoldings', '@getProfiles']);
   },
+  simpleSearchByParameter: (parameter, value) => {
```

Comment:
May be just searchByParameter without simple?

Reply:
searchByParametr with some parametrs in that method explore by Tanya and Dima.

Reply:
does it mean that there are several methods with nearest steps? If yes, can we join them into one method?